packages/manager/modules/pci/src/projects/project/instances/translations/Messages_fr_FR.json

```diff
@@ -33,8 +33,10 @@
   "pci_projects_project_instances_status_BUILD": "Création",
   "pci_projects_project_instances_status_SHUTOFF": "Éteinte",
   "pci_projects_project_instances_status_RESCUE": "Rescue",
-  "pci_projects_project_instances_status_SHELVED": "Shelved",
-  "pci_projects_project_instances_status_SHELVED_OFFLOADED": "Shelved offloaded",
+  "pci_projects_project_instances_status_SHELVED": "Suspendue",
```

Comment:
Same translation than for `SHELVED_OFFLOADED` ?

Reply:
Yes @marie-j, PCI team ask to translate it also.